### DISCUSSION TITLE
What do you think of the `rich_interactive` package name? Would you change it?

### DISCUSSION THREAD

**Discussion by noisy** [noisy/rich_interactive#2] [General]
Unfortunately, [richer](https://pypi.org/project/richer/) is already taken.

While `rich_interactive` is somewhat descriptive, it definitely lacks appeal. What do you think? What name should this package have?

**Comment by lukmarcus** [comment]
- richmore
- richard 
- richarder (rich + harder)
- richteractive (rich + interactive)
